content/blog/announcing-crossguard-preview/index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Announcing Crossguard Preview"
+date: 2019-12-02
+draft: true
+meta_desc: "Today we are announcing Pulumi CrossGuard, a Policy as Code solution, is available all Team Pro and Enterprise customers to preview."
+meta_image: meta.png
+authors: ["erin-krengel"]
+tags: ["crossguard", "policy-as-code", "New-Features", "Pulumi-News", "Features"]
+---
+
+Over the past few months, we have been hard at work on Pulumi CrossGuard, a Policy as Code solution. Using the `pulumi-policy` library, you can express flexible business and security rules using code. CrossGuard enables organization administrators to enforce these policies across their organization or just on specific stacks. CrossGuard allows you to verify or enforce custom policies on changes before they are applied to your resources.
+
+<!--more-->
+
+You can run policies against any Pulumi stack, independent of the stack's language. This means you can have policies written in TypeScript and run them against a Python-based stack. Policies are grouped in *Policy Packs* and run against a stack during both `pulumi preview` and `pulumi up`. Each resource in a stack is run through the Policy Pack prior to creation, meaning you can block the creation of out-of-compliance resources.
+
+## Why
+
+Using Pulumi, developers and operators are empowered to self-provision their infrastructure and move quickly. One of our core goals is to allow you to codify best practices and share them with your team. We often hear from organization leaders that they are concerned about enforcing security and compliance best-practices. CrossGuard provides a way to automatically enforce compliance, allowing teams to operate self-sufficiently while still ensuring organization-wide requirements are met.
+
+## Key Features
+
+The key features available during the CrossGuard preview are:
+
+* TypeScript [pulumi-policy SDK](https://github.com/pulumi/pulumi-policy) to express custom policies
+* [Run Policy Packs locally](https://www.pulumi.com/docs/get-started/policy-as-code/authoring-a-policy-pack/#testing-the-policy-pack-locally) using the `--policy-pack` flag
+* [AWSGuard library](https://github.com/pulumi/pulumi-awsguard) - codifies best practices for AWS
```

Comment:
Or alternatively "@pulumi/awsguard package" (depending on what the first bullet is).